src/components/board/Board.vue

```diff
@@ -124,13 +126,51 @@ export default {
 		},
 	},
 	watch: {
-		id: 'fetchData',
+		id(newValue, oldValue) {
+			if (oldValue) {
+				// close old session
+				sessionApi.closeSession(oldValue, this.token)
```

Comment:
Would it make sense to wait for the async function before initialising the new session here to avoid race conditions?

Reply:
in 8f698e8c015a7c98898a514c7f0fd31203079d7d i've changed now the way sessions are created and closed, which should avoid any potential race condition :)